app/_src/kubernetes-ingress-controller/guides/configuring-health-checks.md

```diff
@@ -2,355 +2,301 @@
+1. Test the Ingress rule by sending 2 requests that represent a failure from upstream and then send a request for 200. The requests with `/status/500` simulate a failure from upstream.
+    Send two requests with `staus/500`.
+    ```bash
+    $ curl -i $PROXY_IP/test/status/500
+    $ curl -i $PROXY_IP/test/status/500
+    ```
+    The results should look like this:
+    ```text
+    HTTP/1.1 500 INTERNAL SERVER ERROR
+    Content-Type: text/html; charset=utf-8
+    Content-Length: 0
+    Connection: keep-alive
+    Server: gunicorn/19.9.0
+    Access-Control-Allow-Origin: *
+    Access-Control-Allow-Credentials: true
+    X-Kong-Upstream-Latency: 2
+    X-Kong-Proxy-Latency: 0
+    Via: kong/3.1.1
+    ```
+    Send the third request with `staus/200`
+    ```bash
+    $ curl -i $PROXY_IP/test/status/200
+    ```
+    The results should look like this:
+    ```text
+    HTTP/1.1 200 OK
+    Content-Type: text/html; charset=utf-8
+    Content-Length: 0
+    Connection: keep-alive
+    Server: gunicorn/19.9.0
+    Access-Control-Allow-Origin: *
+    Access-Control-Allow-Credentials: true
+    X-Kong-Upstream-Latency: 2
+    X-Kong-Proxy-Latency: 0
+    Via: kong/3.1.1
+    ```            
+    Kong has not short-circuited because there were only two failures.
+
+1. Send 3 requests to open the circuit, and then send a normal request.
+    Send two requests with `staus/500`. 
+    ```bash
+    $ curl -i $PROXY_IP/test/status/500
+    $ curl -i $PROXY_IP/test/status/500
+    $ curl -i $PROXY_IP/test/status/500
+    ```
+    The results should look like this:
+    ```text
+    HTTP/1.1 500 INTERNAL SERVER ERROR
+    Content-Type: text/html; charset=utf-8
+    Content-Length: 0
+    Connection: keep-alive
+    Server: gunicorn/19.9.0
+    Access-Control-Allow-Origin: *
+    Access-Control-Allow-Credentials: true
+    X-Kong-Upstream-Latency: 2
+    X-Kong-Proxy-Latency: 0
+    Via: kong/3.1.1
+    ```          
+    Send the fourth request with `staus/200`
+    ```bash
+    $ curl -i $PROXY_IP/test/status/200
+    ```
+    The results should look like this:
+    ```text
+    HTTP/1.1 503 Service Temporarily Unavailable
+    Content-Type: application/json; charset=utf-8
+    Connection: keep-alive
+    Content-Length: 62
+    X-Kong-Response-Latency: 1
+    Server: kong/3.3.1
+    
+    {
+      "message":"failure to get a peer from the ring-balancer"
+    }%                  
+    ```            
+
+    Kong returns a 503, indicating that the service is unavailable. Because there is only one Pod of `httpbin` service running in the cluster, and that is throwing errors, Kong does not proxy anymore requests.
+
+There are a few options:
+
+- Delete the current `httpbin` Pod; Kong then sends proxy requests to the new
+Pod that comes in its place.
+- Scale the httpbin deployment; Kong then sends proxy requests to the new
+  Pods and leave the short-circuited Pod out of the loop.
+- Manually change the Pod health status in Kong using Kong's Admin API.
```

Comment:
Which admin API endpoint should I use?

Reply:
> Which admin API endpoint should I use?

Can you please help me in answering that question. I only formatted that bit.

Reply:
https://docs.konghq.com/gateway/latest/admin-api/#set-target-as-healthy

Reply:
@mheap  should we remove this bullet point because we just disabled the GatewayAPI in the install guide?

Reply:
Done!